### DISCUSSION TITLE
How to click the element of `input[type=button]` ?

### DISCUSSION THREAD

**Discussion by peterroe** [getgauge/taiko#2564] [General]
I want to click the button element to jump other website：

```html
<input type="button"  class="go_next" onclick="go(ee)">
  Go !
</input>
```

but how can I click the element?

I try to such :

```shell
click(Button({class: "go_next"}))
```

But it can't work... help :(

**Comment by ravikiranhebbar88** [comment]
Hi peterroe 

Try with

 **await click($(".go_next"));**

**Reply by peterroe** [reply]
Thank you. But it can't work, too. Because lots of elements I get if i run:

 `document.querySelectorAll('.go_next')`

class cannot determine uniqueness.

the real element: 

```html
<input name="0MKKey" class="edit_lobo_cell"  type="button" value="login" data-localize="pc.15790027008931" onclick="&nbsp;ee(1)">
```

I try to run `$(input.edit_lobo_cell[type="button"])` to get the only it. but it can't work, too

**Comment by gulshankamboj** [comment]
click($("//input[@class='go_next']")) 
or 
click($("//input[@class='go_next']"),{force:true}) 

Let me know if it works here!!

**Reply by peterroe** [reply]
Thanks :), I have sovled the problem.
